phpcs-ruleset.xml

```diff
@@ -20,13 +20,14 @@
     <exclude-pattern>profiles/multisite_drupal_communities/inject_data.php</exclude-pattern>
     <exclude-pattern>profiles/multisite_drupal_communities/themes/bootstrap</exclude-pattern>
     <exclude-pattern>profiles/multisite_drupal_standard/themes/bootstrap</exclude-pattern>
-    <exclude-pattern>profiles/multisite_drupal_standard/themes/europa</exclude-pattern>
+    <exclude-pattern>profiles/multisite_drupal_standard/themes/ec_europa</exclude-pattern>
 
     <rule ref="./vendor/drupal/coder/coder_sniffer/Drupal" />
 
     <!-- Exclude third party code. -->
     <exclude-pattern>profiles/common/libraries/</exclude-pattern>
     <exclude-pattern>profiles/common/modules/custom/ecas/libraries/</exclude-pattern>
+    <exclude-pattern>profiles/common/themes/ec_europa</exclude-pattern>
```

Comment:
And what about `profiles/common/themes/ec_resp`? Should we not exclude it as as we do with ec_europa?